<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSpline</class>
 <widget class="QDialog" name="AddSpline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1898</width>
    <height>1367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1633</x>
     <y>1300</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xBox">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>1172</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="yBox">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>1210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="zBox">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>1250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>1180</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>1220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>1250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="scaleBox">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>1132</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>1140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>1729</x>
     <y>1071</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить точку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeSplineButton">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>930</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="splineNumBox">
   <property name="geometry">
    <rect>
     <x>1821</x>
     <y>1031</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>1040</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Spline</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>980</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddSpline</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddSpline</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
